### PR TITLE
Created a new troubleshooting guide applicable to OCP 4.3, 4.4 and 4.5.

### PR DESCRIPTION
**Pull request by johnwilkins**
@ahardin-rh This troubleshooting guide is applicable to 4.3, 4.4 and 4.5. I will send another PR fter this is merged containing a module that is for 4.3 only.

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@johnwilkins @ahardin-rh we don't yet have a webscale site for 4.5. Is that still expected for the 15th considering that OCP 4.5 is delayed?

**Comment by johnwilkins** [comment]
It's not expected to be published for 4.5 if it is delayed. Can it still be merged to master and cherry picked to 4.4 and 4.3 at least? If it's cherry picked to 4.5 as well, I'm guessing that wouldn't be a problem since it doesn't get published until there is a webscale site for 4.5. Correct?

**Comment by ahardin-rh** [comment]
@johnwilkins Correct. I can cherry-pick it to 4.5 so that it publishes when 4.5 is GA.

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc [hunk 1]
```diff
@@ -5,10 +5,38 @@ include::modules/common-attributes.adoc[]
...
-// For general, {product-title} troubleshooting, see <xref>.
+.Troubleshooting the installer workflow
 
-include::modules/ipi-install-troubleshooting-the-bootstrap-vm.adoc[leveloffset=+1]
+Prior to troubleshooting the installation environment, it is critical to understand the overall flow of the IPI installation on bare metal. The diagrams below provide a troubleshooting flow with a step-by-step breakdown for the environment.
 
-include::modules/ipi-install-troubleshooting-the-control-plane.adoc[leveloffset=+1]
+image:flow1.png[Flow-Diagram-1]
+
+_Workflow 1 of 4_ illustrates a troubleshooting workflow when the `install-config.yaml` file has errors or the RHCOS images are inaccessible.  Troubleshooting suggestions can be found at <<install_config,Troubleshooting `install-config.yaml`>>.
```
Comment (ahardin-rh) [inline installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:15]: Looks like this link is broken. We don't use this link format. It must be in xref: format per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-internal-content

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc [hunk 2]
```diff
@@ -5,10 +5,38 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following troubleshooting sections address Installer Provisioned Infrastructure (IPI) troubleshooting.
 
-// For general, {product-title} troubleshooting, see <xref>.
+.Troubleshooting the installer workflow
```
Comment (ahardin-rh) [inline installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:9]: To ensure this is area of content is included in the TOC, we should use this heading format instead:

= Troubleshooting the installer workflow

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc [hunk 3]
```diff
@@ -5,10 +5,38 @@ include::modules/common-attributes.adoc[]
...
+Prior to troubleshooting the installation environment, it is critical to understand the overall flow of the IPI installation on bare metal. The diagrams below provide a troubleshooting flow with a step-by-step breakdown for the environment.
 
-include::modules/ipi-install-troubleshooting-the-control-plane.adoc[leveloffset=+1]
+image:flow1.png[Flow-Diagram-1]
+
+_Workflow 1 of 4_ illustrates a troubleshooting workflow when the `install-config.yaml` file has errors or the RHCOS images are inaccessible.  Troubleshooting suggestions can be found at <<install_config,Troubleshooting `install-config.yaml`>>.
+
+image:flow2.png[Flow-Diagram-2]
+
+_Workflow 2 of 4_ illustrates a troubleshooting workflow for <<ipi-install-troubleshooting-bootstrap-vm_{context}, bootstrap VM issues>>, <<ipi-install-troubleshooting-bootstrap-vm-cannot-boot_{context}, bootstrap VMs that cannot boot up the cluster nodes>>, and  <<ipi-install-troubleshooting-bootstrap-vm-inspecting-logs_{context}, inspecting logs>>.
```
Comment (ahardin-rh) [inline installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:19]: All internal links should be in xref format; looks like these are used throughout

File: installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc (hunk 1 above)
Comment (ahardin-rh) [inline installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc:15]: s/RHCOS/{op-system-first}

We have the RHCOS product name defined in the common-attributes.adoc file. Use {op-system-first} for the first instance and then {op-system} for all following RHCOS references.

File: modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+[core@localhost ~]$ sudo podman logs -f <container-name>
+----
++
+Replace `<container-name>` with one of `ironic-api`, `ironic-conductor`, or `ironic-inspector`. If you encounter an issue where the master nodes are not booting up via PXE, check the `ironic-conductor` pod. The `ironic-conductor` pod contains the most detail about the attempt to boot the cluster nodes, because it attempts to log in to the node over IPMI.
+
+.Potential reason
+The cluster nodes might be in the `ON` state when deployment started.
+
+.Solution
+Make sure to power off the OpenShift cluster nodes before you begin the
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc:40]: s/OpenShift/{product-title}

File: modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-bootstrap-vm-cannot-boot_{context}"]
+= Bootstrap VM cannot boot up the cluster nodes
+
+During the deployment, it is possible for the bootstrap VM to fail to boot the cluster nodes, which prevents the VM from provisioning the nodes with the RHCOS image. This scenario can arise due to:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc:7]: s/RHCOS/{op-system}

File: modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+----
+
+. To check the container logs, execute the following:
++
+[source,bash]
+----
+[core@localhost ~]$ sudo podman logs -f <container-name>
+----
++
+Replace `<container-name>` with one of `ironic-api`, `ironic-conductor`, or `ironic-inspector`. If you encounter an issue where the master nodes are not booting up via PXE, check the `ironic-conductor` pod. The `ironic-conductor` pod contains the most detail about the attempt to boot the cluster nodes, because it attempts to log in to the node over IPMI.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc:34]: s/pod/Pod (several instances)

File: modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc (hunk 4 above)
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc:40]: s/Make sure to power off/Power off

File: modules/ipi-install-troubleshooting-install-config.adoc [hunk 7]
```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-install-config"]
+
+= Troubleshooting `install-config.yaml`
+
+The `install-config.yaml` configuration file represents all of the nodes that are part of the OpenShift cluster. The file contains the necessary options consisting of but not limited to `apiVersion`, `baseDomain`, `imageContentSources` and virtual IP addresses. If errors occur early in the deployment of the OpenShift cluster, the errors are likely in the `install-config.yaml` configuration file.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-install-config.adoc:8]: s/OpenShift/{product-title} 
(several instances)

File: modules/ipi-install-troubleshooting-install-config.adoc [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+.Procedure
+
+. Use the guidelines in https://www.redhat.com/sysadmin/yaml-tips[YAML-tips].
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-install-config.adoc:12]: link:https://www.redhat.com/sysadmin/yaml-tips[YAML-tips]

File: modules/ipi-install-troubleshooting-install-config.adoc [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+. Verify the YAML syntax is correct using http://www.yamllint.com/[syntax-check].
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-install-config.adoc:13]: link:http://www.yamllint.com/[syntax-check]

File: modules/ipi-install-troubleshooting-install-config.adoc [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+. Verify the RHCOS QEMU images are properly defined and accessible via
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-install-config.adoc:14]: s/RHCOS/{op-system}

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 11]
```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-bootstrap-vm_{context}"]
+
+= Bootstrap VM issues
+
+The OpenShift installer spawns a bootstrap node virtual machine, which handles provisioning the OpenShift cluster nodes.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:8]: s/OpenShift/{product-title}

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+----
++
+----
+ Id    Name                           State
+ --------------------------------------------
+ 12    openshift-xf6fq-bootstrap      running
+----
+
+
+NOTE: The name of the bootstrap VM is always the clustername, followed
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:26]: s/clustername/cluster name

followed by a 

    s/bootstrap/`bootstrap`

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+by random set of characters and ending in the word bootstrap.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:27]: s/bootstrap/`bootstrap`

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+[kni@provisioner ~]$ sudo virsh list
+----
++
+----
+ Id    Name                           State
+ --------------------------------------------
+ 12    openshift-xf6fq-bootstrap      running
+----
+
+
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:25]: Notes are formatted as:

[NOTE]
====
Text for admonition
====

per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+. Verify `libvirtd` is running on the system:
++
+[source,bash]
+----
+[kni@provisioner ~]$ systemctl status libvirtd
+----
++
+----
+● libvirtd.service - Virtualization daemon
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm.adoc:39]: Is the ● intentional?

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+When experiencing issues downloading or accessing the RHCOS images, first verify that the URL is correct in the `install-config.yaml` configuration file.
+
+.Example of internal webserver hosting RHCOS images
+[source,yaml]
+----
+bootstrapOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-qemu.x86_64.qcow2.gz?sha256=9d999f55ff1d44f7ed7c106508e5deecd04dc3c06095d34d36bf1cd127837e0c
+clusterOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-openstack.x86_64.qcow2.gz?sha256=a1bda656fa0892f7b936fdc6b6a6086bddaed5dafacedcd7a1e811abb78fe3b0
+----
+
+The `ipa-downloader` and `coreos-downloader` containers download resources from a webserver or the external `https://quay.io` registry, whichever is specified in the `install-config.yaml` configuration file. Verify the following two containers are up and running and inspect their logs as needed:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc:16]: If you don't want quay.io to be a hyperlink, use this format:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#example-urls

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+* `ipa-downloader`
+* `coreos-downloader`
+
+.Procedure
+
+. Login to the bootstrap VM:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc:23]: s/Login/Log in

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+----
+[core@localhost ~]$ journalctl -xe
+----
++
+[source,bash]
+----
+[core@localhost ~]$ journalctl -b -f -u bootkube.service
+----
+
+. Verify all the pods, including `dnsmasq`, `mariadb`, `httpd`, and `ironic`, are running:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc:56]: s/pods/Pods

File: modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
++
+[source,bash]
+----
+[core@localhost ~]$ sudo podman ps
+----
+
+. If there are issues with the pods, check the logs of the containers with issues. To check the log of the `ironic-api`, execute the following:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc:63]: s/pods/Pods

File: modules/ipi-install-troubleshooting-cluster-nodes-will-not-pxe.adoc [hunk 20]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-cluster-nodes-will-not-pxe_{context}"]
+
+= Cluster nodes will not PXE boot
+
+When OpenShift cluster nodes will not PXE boot, execute the following checks on the cluster nodes that will not PXE boot.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-cluster-nodes-will-not-pxe.adoc:8]: s/OpenShift/{product-title}

File: modules/ipi-install-troubleshooting-cluster-nodes-will-not-pxe.adoc [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+
+.Procedure
+
+. Check the network connectivity to the `provisioning` network.
+
+. Ensure PXE is enabled on the NIC for the `provisioning` network and PXE is  disabled for all other NICs.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-cluster-nodes-will-not-pxe.adoc:14]: is disabled (possible extra space)

File: modules/ipi-install-troubleshooting-api-not-accessible.adoc [hunk 22]
```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-api-not-accessible_{context}"]
+
+= API not accessible
+
+When the cluster is running and clients cannot access the API, domain name resolution issues may impede access to the API.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-api-not-accessible.adoc:8]: s/may/might

File: modules/ipi-install-troubleshooting-api-not-accessible.adoc [hunk 23]
```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-api-not-accessible_{context}"]
+
+= API not accessible
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-api-not-accessible.adoc:6]: The API is not accessible

File: modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 24]
```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-cleaning-up-previous-installations_{context}"]
+= Cleaning up previous installations
+
+In the event of a previous failed deployment, remove the artifacts from the failed attempt before attempting to deploy OpenShift again.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:7]: s/OpenShift/{product-title}

File: modules/ipi-install-troubleshooting-registry-issues.adoc [hunk 25]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-registry-issues_{context}"]
+
+= Issues with creating the registry
+
+When creating a disconnected registry, you may encounter a "User Not Authorized" error when attempting to mirror the registry. This error may occur if you fail to append the new authentication to the existing `pull-secret.txt` file.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-registry-issues.adoc:8]: s/may/might

    s/"User Not Authorized"/`User Not Authorized`

File: modules/ipi-install-troubleshooting-registry-issues.adoc [hunk 26]
```diff
@@ -0,0 +1,42 @@
...
+[user@registry ~]$ /usr/local/bin/oc adm release mirror \
+  -a pull-secret-update.json
+  --from=$UPSTREAM_REPO \
+  --to-release-image=$LOCAL_REG/$LOCAL_REPO:${VERSION} \
+  --to=$LOCAL_REG/$LOCAL_REPO
+----
++
+[NOTE]
+====
+Example output of the variables used to mirror the install images
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-registry-issues.adoc:25]: images:

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 27]
```diff
@@ -0,0 +1,265 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-misc-issues_{context}"]
+
+= Miscellaneous issues
+
+== Addressing the `runtime network not ready` error
+
+After the deployment of a cluster you may receive the following error:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:10]: s/may/might

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 28]
```diff
@@ -0,0 +1,265 @@
...
+
+----
+`runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: Missing CNI default network`
+----
+
+The cluster network operator is responsible for deploying the networking components in response to a special object created by the installer. It runs very early in the installation process, after the Control Plane (master) nodes have come up but before the bootstrap control plane has been torn down. It can be indicative of more subtle installer issues, such as long delays in bringing up Control Plane (master) nodes or issues with `apiserver` communication.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:16]: Cluster Network Operator

Comment (ahardin-rh) [reply modules/ipi-install-troubleshooting-misc-issues.adoc:16]: up, but before

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 29]
```diff
@@ -0,0 +1,265 @@
...
+
+.Procedure
+
+. Inspect the pods in the `openshift-network-operator` namespace:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:20]: s/pods/Pods

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 30]
```diff
@@ -0,0 +1,265 @@
...
+----
+
+. Retrieve the logs:
++
+[source,bash]
+----
+[kni@provisioner ~]$ kubectl -n openshift-network-operator logs -l "name=network-operator"
+----
++
+On high availability clusters with three or more Control Plane (master) nodes, the operator will perform leader election and all other operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:75]: s/operator/Operator
s/operators/Operators

link:https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting]

Also, all links to GitHub repos must be approved by Support.
cc @sferich888 Is this usage in the webscale docs okay?

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 31]
```diff
@@ -0,0 +1,265 @@
...
+
+== Cluster nodes not getting the correct IPv6 address over DHCP
+
+If the cluster nodes are not getting the correct IPv6 address over DHCP, check the following:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:79]: Should we spell out DHCP here, or is this a common enough acronymn?

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 32]
```diff
@@ -0,0 +1,265 @@
...
+. Retrieve the `hostname`:
++
+[source,bash]
+----
+[core@master-X ~]$ hostname
+----
++
+If the hostname is `localhost`, proceed with the following steps.
++
+NOTE: Where `X` is the master node number.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:120]: [NOTE]
    ====
    Note text
    ====

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 33]
```diff
@@ -0,0 +1,265 @@
...
+. Ensure `kubelet` booted with the correct hostname:
++
+[source,bash]
+----
+[core@master-X ~]$ sudo journalctl -fu kubelet.service
+----
+
+If the cluster node is not getting the correct hostname over DHCP after the cluster is up and running, such as during a reboot, the cluster will have a pending `csr`. **Do not** approve a `csr`, or other issues may arise.
+
+.Adderssing a `csr`
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:179]: Addressing

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 34]
```diff
@@ -0,0 +1,265 @@
...
+  },
+  "status": "Failure",
+  "message": "forbidden: User \"system:anonymous\" cannot get path \"/\"",
+  "reason": "Forbidden",
+  "details": {
+  },
+  "code": 403
+----
+
+. Identify the authentication operator errors from the `provisioner` node:
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:249]: s/operator/Operator

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 35]
```diff
@@ -0,0 +1,265 @@
...
+----
++
+[source,bash]
+----
+Event(v1.ObjectReference{Kind:"Deployment", Namespace:"openshift-authentication-operator", Name:"authentication-operator", UID:"225c5bd5-b368-439b-9155-5fd3c0459d98", APIVersion:"apps/v1", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'OperatorStatusChanged' Status for clusteroperator/authentication changed: Degraded message changed from "IngressStateEndpointsDegraded: All 2 endpoints for oauth-server are reporting"
+----
+
+.Solution
+
+. Ensure that the cluster name for every deployment is unique ensuring no conflict.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:263]: unique, ensuring

File: modules/ipi-install-troubleshooting-misc-issues.adoc [hunk 36]
```diff
@@ -0,0 +1,265 @@
...
+
+. Turn off all the rogue nodes which are not part of the cluster deployment that are using the same cluster name. Otherwise, the authentication pod of the  {product-title} cluster may never start successfully.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-misc-issues.adoc:265]: e/pod/Pod

File: modules/ipi-install-troubleshooting-ntp-out-of-sync.adoc [hunk 37]
```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-ntp-out-of-sync_{context}"]
+
+= NTP out of sync
+
+The deployment of OpenShift clusters depends on NTP synchronized clocks among the cluster nodes. Without synchronized clocks, the deployment may fail due to clock drift if the time difference is greater than two seconds.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-ntp-out-of-sync.adoc:8]: s/OpenShift/{product-title}

File: modules/ipi-install-troubleshooting-reviewing-the-installation.adoc [hunk 38]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-reviewing-the-installation_{context}"]
+
+= Reviewing the installation
+
+After installation, ensure the installer deployed the nodes and pods successfully.
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-reviewing-the-installation.adoc:8]: s/pods/Pods

File: modules/ipi-install-troubleshooting-reviewing-the-installation.adoc [hunk 39]
```diff
@@ -0,0 +1,34 @@
...
+
+.Procedure
+
+. When the OpenShift cluster nodes are installed appropriately,
```
Comment (ahardin-rh) [inline modules/ipi-install-troubleshooting-reviewing-the-installation.adoc:12]: s/OpenShift/{product-title}

**Comment by ahardin-rh** [comment]
@johnwilkins Just some minor comments from me. Please let me know if you have any concerns or if you need my help with these changes. Thanks!

File: modules/ipi-install-troubleshooting-bootstrap-vm.adoc (hunk 15 above)
Comment (johnwilkins) [reply modules/ipi-install-troubleshooting-bootstrap-vm.adoc:39]: Yes. It's in the output.

File: modules/ipi-install-troubleshooting-misc-issues.adoc (hunk 31 above)
Comment (johnwilkins) [reply modules/ipi-install-troubleshooting-misc-issues.adoc:79]: It's common enough. We spelled out when it's used in other compound acronyms like DUID, etc.

**Pull request merged**